### PR TITLE
[new] add angular-foundation@0.8.0 with git auto-update

### PR DESCRIPTION
**Pull request by Amomo**
Hi @LeaYeh 

For #4629, I add angular-foundation at v0.8.0 and add git auto-update config in its package.json.
repo: https://github.com/pineconellc/angular-foundation

The author put build files in another repo [angular-foundation-bower](https://github.com/pineconellc/angular-foundation-bower)
So I set the url in auto-update config to point it.

Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `83`, Star `1,003`, Fork `279`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by LeaYeh** [comment]
i think this or is ok.
pls rebase it

**Comment by Amomo** [comment]
@LeaYeh I have rebased it, could you check again? Thank you! :-D

**Comment by Amomo** [comment]
@LeaYeh I have rebased it, could you come to check this PR? Thank you~

**Comment by Amomo** [comment]
@LeaYeh I have rebased this PR, could you come to check it? Thank you~

File: ajax/libs/angular-foundation/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/pineconellc/angular-foundation.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/pineconellc/angular-foundation-bower.git",
+    "basePath": "",
+    "files": [
+      "mm-foundation*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/angular-foundation/package.json]: "mm-foundation*" will be fine

**Pull request merged**